WordPress/Classes/ViewRelated/Tools/SettingsTextViewController.m

```diff
@@ -123,6 +123,8 @@ - (void)viewDidAppear:(BOOL)animated
 {
     [super viewDidAppear:animated];
     [self.textField becomeFirstResponder];
+    // Fire initial status
+    [self validateTextInput:_textField];
```

Comment:
It's probably better to use the property here instead of the instance variable in case we needed to change the getter.

```suggestion
    [self validateTextInput:self.textField];
```